protocols/imap/src/main/java/org/apache/james/imap/main/PathConverter.java

```diff
@@ -29,12 +29,13 @@
 import com.google.common.base.Joiner;
 import com.google.common.base.Preconditions;
 import com.google.common.base.Splitter;
-import com.google.common.base.Strings;
 import com.google.common.collect.Iterables;
 
 public class PathConverter {
 
-    private static final int NAMESPACE = 0;
+    public static final String DELEGATED_MAILBOXES_BASE = "Other users";
```

Comment:
It probably belongs to IMAP protocol, we should not use a string representation as long as we don't require to display things to user

Reply:
The intend of this class is to convert to and from user input. It is also located in imap. I thus don't understand your comment

Reply:
Yes, after several hours of work on that topic, I start to understand.